### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by rnathuji**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+*             @rnathuji @P-Gill97 @MReyna12
+/html/        @mjoss17
+/mbz/         @mjoss17
+/json/        @mjoss17
+/policies/    @mjoss17
```
Comment (rnathuji) [inline .github/CODEOWNERS:5]: @mjoss17 , @P-Gill97 , @MReyna12 - Can y'all take a look at this to confirm it looks right per [the docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)? Devs are default owners for the repo, and PDM is owner of everything in content directories.  Once this PR is merged, this should trigger review requests on PRs based on matches. When we want, we can enable `Require review from Code Owners` for `main` which should then allow PDM to merge content PRs w/o review, but require anyone else to get a review from PDM before they can merge.

Comment (MReyna12) [reply .github/CODEOWNERS:5]: All of the above checks out, but maybe I missed where enabling `Require review from Code Owners` allows the Code Owner to merge PRs w/o review. Is it just inherently understood that if you are a Code Owner of a directory and/or file type, that if you make changes in said directory / file type, then you don't need to approve your own code?

Comment (rnathuji) [reply .github/CODEOWNERS:5]: Ah -- that setting is in the [branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule) that a repo admin needs to toggle. My understanding is that once enabled, a code owner does not need to get a PR :heavy_check_mark: prior to merging (e.g. they would retain the same XP as right now here you can request a review, but you don't have to get an approval to merge).

Comment (MReyna12) [reply .github/CODEOWNERS:5]: I see - I guess I was looking for explicit language that stated if the PR is created by a code owner in the repo they own, then their PR won't require approval prior to merging.

Comment (rnathuji) [reply .github/CODEOWNERS:5]: :thinking: - If we want to confirm, we can enable the flag after merging this PR to see if things behave as expected for PDM and CM (or, I guess we can create test PRs ourselves) :man_shrugging: .

Comment (MReyna12) [reply .github/CODEOWNERS:5]: It could just be my innate paranoia getting in the way, but it might be a good idea to do so 😆

Comment (rnathuji) [reply .github/CODEOWNERS:5]: Let me merge this and try things out.

Comment (rnathuji) [reply .github/CODEOWNERS:5]: It looks like @MReyna12 's paranoia was well placed. The rule to require approval from a code owner seems to apply to code owners as well.  Right now, I cannot merge [this PR](https://github.com/openstax/k12-contents-raise/pull/1539) as a code owner of the corresponding file unless I get approval from one of the other code owners.

**Pull request merged**